webrtc.html

```diff
@@ -2504,6 +2504,31 @@
             </ol>
           </dd>
 
+          <dt>RTCRtpSender createRtpSender (DOMString kind)</dt>
+
+          <dd>
+            <p>Equivalent to
+            calling <a href="#dom-peerconnection-addtrack"><code>addTrack(null)</code></a>,
```

Comment:
I have updated the text to not be worded in terms of addTrack(null).  See if you like it better.